hydra-node/src/Hydra/Chain/Direct.hs

```diff
@@ -164,6 +178,68 @@ mkTinyWallet tracer config = do
     hoistEpochInfo (first show . runExcept) $
       Consensus.interpreterToEpochInfo interpreter
 
+withOfflineChain ::
+  Tracer IO DirectChainLog -> -- TODO(ELAINE): change type maybe ?
```

Comment:
Is it okay to reuse DirectChainLog here, or should we have a different type to indicate this is offline mode?

Reply:
You would need to add specific constructors for the specific loggging you would do in off-line mode?